mycore-base/src/main/resources/xsl/functions/mcrversion.xsl

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="3.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:mcrversion="http://www.mycore.de/xslt/mcrversion"
+                exclude-result-prefixes="fn xs">
+
+    <xsl:function name="mcrversion:getCompleteVersion" as="xs:string">
+        <xsl:variable name="versionDoc" select="fn:document('version:completeVersion')"/>
+        <xsl:value-of select="$versionDoc/version/text()"/>
+    </xsl:function>
+
+    <xsl:function name="mcrversion:revision" as="xs:string">
```

Comment:
do we need a function for every version property?
I would return a version XML with all properties as subelements or attributes and the developer can choose what's needed
mcrversion:versionXML/@version,..., mcrversion:versionXML/@gitDescribe